doc/tutorial.md

```diff
@@ -1191,6 +1192,84 @@ worker_defs {
 
 Please refer to doc [tc.md](tc.md).
 
+<a id='acl'/>
+
+# IPset based ACL
+An access control list (ACL) is a list of access control entries (ACE). Each ACE in an ACL identifies a trustee and specifies the access rights allowed or denied for that trustee. DPVS implements an ACL mechanism based on [IPset module](IPset.md). <br>
+Each service contains an ACL rule list where you can add rules. All rules should follow the **PATTERN** of `set=NAME,type={black|white}[,prio={1-100},dir={ingress|egress}]` where `set` and `type` are **essential**. Then you can dynamically change the ACL by adding/deleting entries in the set. For a 'black-type' rule, entries in the set will not be able to establish a DPVS conn. And if a packet is not contained in any 'white-type' rule, it will also be dropped.<br>
+
+![fnat-two-arm](./pics/fnat-two-arm.png)
+## use ipvsadm
+
+Based on the [Simple Full-NAT (two-arm)](#simple-fnat) example, the following commands demonstrate how to add a blacklist and a whitelist to the service using `ipvsadm`:
+```
+$ dpip ipset add s1 hash:ip
+$ dpip ipset add s2 hash:net
+$ ipvsadm -K -t 10.0.0.100/32 -3 set=s1,type=black,prio=100,dir=ingress
+$ ipvsadm -K -t 10.0.0.100/32 -3 set=s2,type=white,prio=99,dir=ingress
+```
+Now if we check the rules attached to the service:
+```
+$ ipvsadm -Tn -t 10.0.0.100/32 --verbose
+Prot LocalAddress:Port
+TCP  10.0.0.100:80
+  Prio  SetName                         Type            Direction
+  100   s1                              blacklist       ingress
+        Members:
+
+  99    s2                              whitelist       ingress
+        Members:
+```
+We can find that there is no element in the whitelist. So any request to the VIP will fail at DPVS. Let's add the client subnet to the whitelist:
+```
+$ dpip ipset add s2 10.0.0.0/8
+```
+Then request from the subnet 10.0.0.0/8 will succeed. But if you add the client IP to the blacklist, the request will fail again:
```

Comment:
I lelt the blacklist ipset empty, and find didn't set a whitelist ipset，but the service couldn't be accessed.